src/components/App/App.css

```diff
@@ -0,0 +1,9 @@
+.wrap {
+    display: block;
+    margin: auto;
+    width: 300px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5688967)